client/anp/photo/views/search-panel-anp.html

```diff
@@ -0,0 +1,30 @@
+<fieldset ng-controller="ANPSearchPanel as panel">
```

Comment:
Can you use React for it?

Where will it be rendered in the core?

Reply:
I guess I can, it's in https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/search/views/search-parameters.html#L260
will try

Reply:
```
scope.getTemplate = (providerType) => `search-panel-${providerType}.html`;

scope.hasTemplate = (providerType) =>
    providerType != null && $templateCache.get(scope.getTemplate(providerType)) != null;
```

Where do you set the provider type to anp?

Reply:
https://github.com/superdesk/superdesk-anp/pull/1/files#diff-4fecad1efa1f8355a3813827af3a113cR131